packages/imodel-select/src/components/IModelList.scss

```diff
@@ -37,9 +37,9 @@ $imodel-select-cards-border-color: $buic-foreground-disabled;
     border: 1px solid #ccc;
     font-size: 18px;
     cursor: pointer;
-    -webkit-transition: color background-color.2s ease;
-    -moz-transition: color background-color.2s ease;
-    -o-transition: color background-color.2s ease;
+    -webkit-transition: color background-color .2s ease;

```

Comment:
sass/scss should generate the vendor specific prefixes for us when compiled to css. I would actually just get rid of all three properties since we already have a correct `transition` property

Reply:
I'm *pretty* sure that auto-prefixing is what happens, but I don't think sass does it, it might be a postcss plugin the react-script's default webpack configuration, I'll need to double check...

Reply:
Done, thanks!